semana1/portifolio/css/style_port.css

```diff
@@ -0,0 +1,239 @@
+*{
+    margin: 0;
+    padding: 0;
+    scroll-behavior: smooth;
+}
+
+#header{
+    width: 100%;
+    position: fixed;
+    position: -webkit-stiky;
+    top: 0;
+    left: 0;
+}
+
+#menu {
+    background-color: #BDB7A2;
+    height: 40px;
+}
+
+#menu ul{
+    text-align: center;
+}
+
+#menu ul li{
+    margin-top: 10px;
+    display: inline-block;  
+}
+
+#menu ul li a:link, #menu ul li a:visited{
```

Comment:
Cuidado com uso de muitas tags nos combinadores. Se for muito específico, opte por usar `id` ;)